install/kubernetes/helm/istio/values-istio-auth-multicluster.yaml

```diff
@@ -16,6 +16,9 @@ global:
   # Default is 10s second
   refreshInterval: 1s
 
+  # Namespace for ConfigMap which stores clusters configs
+  clusterRegistriesNamespace: istio-system
```

Comment:
We'd want the default to be the same namespace as the other istio control-plane stuff--`.Release.Namespace`

Reply:
@tiswanso yes, I saw you have a PR https://github.com/istio/istio/pull/6446 , and once this merged, I think we do not need this PR?

Reply:
@gyliu513  -- We still need this PR to be able to control this option from helm.  With https://github.com/istio/istio/pull/6446 it just gets taken off the list of multicluster test & doc workflow blockers.